program/index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Выбор программы</title>
+  </head>
+  <body>
+    <main class="main">
+      <section class="header container">
+        <img
+          class="header__logo"
+          src="../img/logo_car_energy.svg"
+          alt="logo_car_energy"
+          width="202"
+          height="60"
+          alt="logo_car_energy"
+        />
+        <ul class="header__menu">
+          <li class="header__item">
+            <a class="header_link" href="#">Главная</a>
+          </li>
+          <li class="header__item">
+            <a class="header_link" href="#">Каталог продукции</a>
+          </li>
+          <li class="header__item">
+            <a class="header_link" href="#">Выбор программы</a>
+          </li>
+        </ul>
+      </section>
+
+      <div class="select-program">
+        <h2 class="select-program__title title">Подбор программы</h2>
+        <form class="" action="" method="GET">
+          <legend class="select-program__decription">
+            Заполните анкету и мы подберем программу питания для вашего кота
+          </legend>
+          <ul class="select-program__data" role="none">
+            <li class="select-program__item">
+              <label class="select-program__text" for="name">Имя:*</label>
+              <input id="name" type="text" name="name" />
+            </li>
+            <li class="select-program__item">
+              <label class="select-program__text" for="weight"
+                >Вес (кг):*</label
+              >
+              <input id="weight" type="text" name="weight" />
+            </li>
+            <li class="select-program__item">
+              <label class="select-program__text" for="age"
+                >Возраст (лет):</label
+              >
+              <input id="age" type="text" name="age" />
+            </li>
+          </ul>
+          <ul class="select-program__data">
+            <li class="select-program__item">
+              <input id="slimming" type="radio" name="target" />
+              <label class="select-program__text" for="slimming"
+                >Похудение</label
+              >
+            </li>
+            <li class="select-program__item">
+              <input id="weight-gain" type="radio" name="target" />
+              <label class="select-program__text" for="weight-gain"
+                >Набор массы</label
+              >
+            </li>
+            <li class="select-program__item">
+              <input id="not-know" type="radio" name="target" />
+              <label class="select-program__text" for="not-know"
+                >Не знаю (Нужен ваш совет)</label
+              >
+            </li>
+          </ul>
+
+          <section class="contacts">
```

Comment:
Please use `fieldset` and `legend` tags instead of `section` and `h3` according to [the article I shared](https://yoksel.github.io/bad-practices/forms-markup/).

Reply:
fixed